spacy/compat.py

```diff
@@ -80,7 +80,7 @@ def getattr_(obj, name, *default):
 
 
 def symlink_to(orig, dest):
-    if is_python2 and is_windows:
+    if (is_python2 or is_python3) and is_windows:
```

Comment:
In this case, I guess we could also just do the following, right?

```suggestion
    if is_windows:
```